### ISSUE TITLE
Installation error

### ISSUE THREAD

**Issue by craftgear** [StanAngeloff/lotte#4]
Hi,
I want to try lotte instead of zombie which can't upload files as binary.

I have an error like following on installation:

```
ERR! Error: ENOENT, no such file or directory '/userhome/.nave/installed/0.6.6/lib/node_modules/lotte/bin/cli.js'
```

I'm using
node -v v0.6.6
npm -v 1.1.0-beta-4

Could you give me any clue?

BTW, why did you name the library 'lotte'?
Do you like to eat this kind of fish?

**Comment by StanAngeloff** [comment]
@craftgear This, indeed, is very odd. I have not come across it. What is the command you execute immediately before this error appears?

I've come across cases where `npm` silently fails if the connection is not stable and some packages fail to download. Try removing/purging the package  and attempt to re-install again using `--force` and `-g` for global.

In the interim, I'll set up a virtual box with the above versions of Node and `npm` to make sure it's not a version issue.

> BTW, why did you name the library 'lotte'?

Lotte uses [Phantom](http://en.wikipedia.org/wiki/The_Phantom_of_the_Opera)JS. The clue is in the novel.

**Comment by StanAngeloff** [comment]
I can confirm this is a version issue, on a fresh Ubuntu box with Node 0.6.6 I get exactly the same error. I would have to investigate, this was last working on `1.0.106`.

**Comment by craftgear** [comment]
Thanks for your quick response.
And now I see where the name came from.

**Comment by StanAngeloff** [comment]
Logged [issue with npm](https://github.com/isaacs/npm/issues/1978), will update on comments.

**Comment by StanAngeloff** [comment]
This has been fixed in the latest Lotte, please use the 0.2.x series. Thanks for reporting this issue.
